frontend/src/api/europepmc.js

```diff
@@ -34,7 +34,7 @@ const searchReferences = async queryIds => {
         refDetails.title
       }</i> ${refDetails.journal}`;
       refs[details.id] = refDetails;
-    } catch {}
+    } catch {} // eslint-disable-line no-empty
```

Comment:
I wonder if there is a better way to solve this than having an empty catch like this and having to disable a warning? Shouldn't we try to catch and somehow react on an error with the get request?

Reply:
I agree that empty `catch` blocks are not a good practice. In this case, the `try` is needed so that no error messages are shown when it fails. Requests failing sometimes is expected here. I think failing and empty responses both lead to the following message that's displayed to the user.

<img width="611" alt="image" src="https://user-images.githubusercontent.com/423498/169068708-635e695f-ef2e-45d6-a1af-1dfb1b80a18d.png">

Reply:
For empty responses I think the `There are no associated references` makes sense. If the requested failed due to for example a server problem, then that message is not necessarily true. It would be bad UX IMO to trigger an error message since this is not the core functionality of the gem browser, but maybe one could display something like `Not possible to access Europe PMC right now` or similar

Reply:
Good idea, my suggestion would be to have that as a new issue.

Reply:
I took another look at this implementation and realized that I was sloppy and misunderstood it before. I thought the data fetching was a part of the `try` block but the actual purpose seems to be to avoid the explicit handling of errors from data processing. For example, an error would be thrown if `details.authorList` would be `null` or `undefined`. This is bad practice indeed and I have created a new issue: https://github.com/MetabolicAtlas/MetabolicAtlas/issues/996

Sorry for not giving this more attention initially and thanks for the discussion!